### PR TITLE
Restructure imagestream files

### PR DESCRIPTION
**Pull request by yselkowitz**
Neither CentOS7 nor UBI8 have been updated to 5.30 yet.

This reverts commit 07fd05eb722269723038bb66cf4bc509c9e00c4d.

### CODE REVIEW

**Comment by phracek** [comment]
[test][test-openshift]

File: imagestreams/perl-centos.json [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        "from": {
+          "kind": "DockerImage",
+          "name": "docker.io/centos/perl-530-centos7:latest"
+        },
+        "referencePolicy": {
+          "type": "Local"
+        }
+      },
+      {
+        "name": "5.26-ubi8",
```
Comment (phracek) [inline imagestreams/perl-centos.json:72]: This should be a part of `imagestreams/perl-rhel.json`, right?

Comment (yselkowitz) [reply imagestreams/perl-centos.json:72]: No, we are adding UBI8 to both RHEL (really, OCP) and CentOS (really, OKD) imagestreams.

**Comment by hhorak** [comment]
Tests failure due to https://github.com/sclorg/container-common-scripts/pull/167, merging now, but will open a new PR with an updated common sub-module to make the tests be green again.

**Comment by hhorak** [comment]
> Tests failure due to [sclorg/container-common-scripts#167](https://github.com/sclorg/container-common-scripts/pull/167), merging now, but will open a new PR with an updated common sub-module to make the tests be green again.

https://github.com/sclorg/s2i-perl-container/pull/184

**Pull request merged**